WICS-UCI/venushacks
# Got the project working on Node v16, cleaned up config files.

## Summary
- VenusHacks site now works with Node v16. For this, I just needed to swap out the deprecated `node-sass` package for `sass`.
- Removed unnecessary netlify configurations. We don't use netlify any more, since we switched to gh-pages.
- Updated npm packages to get rid of security vulnerabilities
- Updated how we compute relative paths for importing components in files. Before we used `.env` but that approach is deprecated and no longer recommended. Now we use `jsconfig.json`.
- Update `manifest.json` to use the VenusHacks name instead of the default "React App" name.

## Test Plan
- Run `npm start` for the website on both node v14 and node v16
- Verify that the site loads on localhost:3000

## Issue(s)
Closes #137 

## Future Followup
When running `npm start`, additional information from webpack is output. 
```
assets by path static/media/*.jpeg 5.93 MiB
  asset static/media/areeta.0f19d065ebba74d087b4.jpeg 2.43 MiB [emitted] [immutable] [from: src/assets/images/speaker-photos/areeta.jpeg] (auxiliary name: main)
  + 50 assets
assets by path static/media/*.png 1.37 MiB
  asset static/media/ai.b0fdd056e9a1e8e9f42d.png 186 KiB [emitted] [immutable] [from: src/assets/images/partners/ai.png] (auxiliary name: main)
  + 31 assets
...
```
This info isn't really necessary but I couldn't figure out how to silence it. We should probably try to get rid of it in the future.
